scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -197,6 +198,10 @@ class Job(val args: Args) extends FieldConversions with java.io.Serializable {
       .toMap.toMap[AnyRef, AnyRef] // linter:ignore the second one is to lift from String -> AnyRef
   }
 
+  def reflectedClasses: Set[Class[_]] = if (args.boolean("scalding.nojobclassreflection")) Set.empty else {
```

Comment:
Actually now that I look at it, I'm not sure that makes sense? the `Config` params are for stuff that go into the hadoop config. This is just being read from the `Args`. Other things being read from args in the `Job` (eg `scalding.nocounters`, or `scalding.flowstats`) aren't referenced in the config. 

Agreed I should add documentation somewhere, just not sure if Config seems like the right place